src/App.scss

```diff
@@ -1 +1,77 @@
-// styles go here
+body {
+  background-image: url("../src/images/wall2.jpg");
+}
```

Comment:
Use type selectors only for reset

Reply:
I was just lazy with this one :) but you are right and I've changed it)